### PR TITLE
feat(web-app): update landing page to new design

### LINKED ISSUE #905: Update home page for v1
**Summary**

This is a placeholder issue to be populated after #883 is completed. There will likely be banner and copy changes, as well as adding another featured library.

**Acceptance criteria**

What requirements must be met to complete this request?

1. To do

### PR DESCRIPTION
**Pull request by alisonjoseph**
Closes #905

Design issue #883
https://www.figma.com/file/M1UolPMDngdACDMMLU7WpX/v1_landing_pg_redlines?node-id=0%3A1

#### Changelog

- Hero
  - Add theme prop
  - Update styles for new homepage image
- P
  - Add large prop 
 - New homepage content
- Add designing and dev arrows to icons package (and to sketch file)

**Fixes**

- Update paragraphs, lists, headings to span 8 columns at medium breakpoint (not 6)
- Fix menu button width on mobile
- Various spacing updates to markdown.scss for heading components
- FeatureCard
  - Style fix to ensure smaller card was on grid
- Divider
  - Add styles for empty component 
  - Fix, should not hang off grid anymore

#### Testing / reviewing

Check homepage at all breakpoints

http://localhost:3000/

**TODO:**
- [x] Add updated mobile hero image from @jeanservaas 

<details>
  <summary>Screenshots</summary>

screencapture-localhost-3000-2022-08-24-09_09_56

</details>

### CODE REVIEW

**Review by aubrey-oneal — CHANGES_REQUESTED** [review]
- [x] Revert 'Easier management' text to original text
Screen Shot 2022-08-19 at 2 06 25 PM

- [x] Change Designing description text to: Learn how design kits are used to build consistent, scalable user interfaces — and get started with your design tool of choice, including Figma, Sketch, Adobe XD, or Axure.

- [x] Change Developing description text to: Learn about libraries and access everything you need to get up and running with your preferred framework —  React, Angular, Web Components, Vue, or Svelte.

**Comment by alisonjoseph** [comment]
@aubrey-oneal updated 
Screen Shot 2022-08-19 at 2 25 41 PM
Screen Shot 2022-08-19 at 2 25 56 PM

File: services/web-app/pages/index.js [hunk 1]
```diff
@@ -18,270 +20,305 @@ import {
...
-      </Grid>
+
+      <Divider />
+
+      <P className={styles['intro-paragraph']}>
+        By standardizing and surfacing our assets, the new site helps makers Find assets that{' '}
+        <mark>comply</mark> with platform requirements, are <mark>convenient</mark> to implement,
+        and are <mark>consistent</mark> with design patterns across the company.
+        <br />↓
+      </P>
```
Comment (francinelucca) [inline services/web-app/pages/index.js:248]: this paragraph column span doesn't seem the same as the redlines at md and >lg breakpoints 
image
image

Comment (francinelucca) [reply services/web-app/pages/index.js:248]: Same happens with the first page paragraph

File: services/web-app/components/hero/hero.module.scss [hunk 2]
```diff
@@ -46,15 +56,20 @@ $hero-height: convert.rem(560px);
...
 
   @include breakpoint.breakpoint(md) {
-    background: url(../../pages/index/images/assets-hero.png) no-repeat top right / auto
-      $hero-height;
+    background: theme.$background url(../../pages/index/images/assets-hero.png) no-repeat top right /
+      auto $hero-height-sm;
+  }
+
+  @include breakpoint.breakpoint(lg) {
+    background: theme.$background url(../../pages/index/images/assets-hero.png) no-repeat top right /
```
Comment (francinelucca) [inline services/web-app/components/hero/hero.module.scss:72]: image is on top of text at lg breakpoint 
image

**Review by francinelucca — CHANGES_REQUESTED** [review]
looking good! found a couple things

File: services/web-app/pages/index.js (hunk 1 above)
Comment (alisonjoseph) [reply services/web-app/pages/index.js:248]: Good catch, I'm checking with design to make sure this page should deviate from our standard column widths.

File: services/web-app/components/hero/hero.module.scss (hunk 2 above)
Comment (alisonjoseph) [reply services/web-app/components/hero/hero.module.scss:72]: Just added the latest images from Jeannie, there is a slight overlap right before it switches from lg to medium, but should be ok.

File: services/web-app/pages/index.js (hunk 1 above)
Comment (alisonjoseph) [reply services/web-app/pages/index.js:248]: Confirmed column width with Jeannie, she updated Figma. Should be good 👍

**Review by francinelucca — APPROVED** [review]
✨

**Review by aubrey-oneal — APPROVED** [review]

**Review by jeanservaas — COMMENTED** [review]
Looks great! I just noticed two things

Could we try to keep the pictograms (in the catalog section) at 64px across ALL breakpoints and see how that looks. Mike noticed that in a lot of places they're getting really tiny.

image

Last thing, it looks like there's a weird hard return or something on the line “Explore the library catalog for all options” You can see it in Large, Tablet and mobile.


image


image

**Pull request merged**